package.json

```diff
@@ -22,15 +22,17 @@
     "core"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=14"
   },
   "main": "dist/sn-table.js",
   "scripts": {
-    "build": "rollup -c rollup.config.js && rollup -c rollup-core.config.js && shx cp assets/* dist",
+    "build": "yarn run locale:generate && rollup -c rollup.config.js && rollup -c rollup-core.config.js && shx cp assets/* dist",
```

Comment:
Why not include this in the rollup config?
Now this will only run on the proper build and not in the watcher...

Reply:
I do not think it should run in the watcher. watcher is for files change in /src. IMO Explicitly run locale:generate to work on the translated string is better.